adafruit_httpserver.py

```diff
@@ -361,3 +361,25 @@ def poll(self):
                 # connection reset by peer, try again later.
                 return
             raise
+
+    @property
+    def requestbuffersize(self) -> int:
```

Comment:
Snake case will make this a little more readable:

```suggestion
    def request_buffer_size(self) -> int:
```

Same thing for the setter of course!

Reply:
I like the snake case and it's definitely easier to read.  I wasn't sure if there were naming standards, so I just mirrored the other function names.  I come from a camel case background, but that doesn't seem to apply in python except in class names.

Reply:
Yup!  You've got it spot on.  I don't think I've ever seen it explicitly about the project, but generally we follow the [PEP8 style guide](https://peps.python.org/pep-0008/) (unless it conflicts with existing CPython API, like `logging`).